python/tests/experimental/constraints_generation/test_constraints_generation.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import pytest
+
+import whylogs as why
+from whylogs.core.constraints import ConstraintsBuilder
+from whylogs.core.metrics.condition_count_metric import (
+    Condition,
+    ConditionCountConfig,
+    ConditionCountMetric,
+)
+from whylogs.core.relations import Predicate
+from whylogs.core.resolvers import STANDARD_RESOLVER, MetricSpec, ResolverSpec
+from whylogs.core.schema import DeclarativeSchema
+from whylogs.experimental.constraints_generation import (
+    generate_constraints_from_reference_profile,
+)
+
+
+@pytest.fixture
+def reference_profile_view():
+    data = {
+        "animal": ["cat", "hawk", "snake", "cat", "mosquito"],
+        "legs": [4, 2, 0, 4, 6],
+        "weight": [4.3, 1.8, 1.3, 4.1, 5.5e-6],
+    }
+    df = pd.DataFrame(data)
+
+    def even(x) -> bool:
+        return x % 2 == 0
+
+    regex_conditions = {"legs_even": Condition(Predicate().is_(even))}
+
+    legs_spec = ResolverSpec(
+        column_name="legs",
+        metrics=[
+            MetricSpec(
+                ConditionCountMetric,
+                ConditionCountConfig(conditions=regex_conditions),
+            ),
+        ],
+    )
+    schema = DeclarativeSchema(STANDARD_RESOLVER)
+    schema.add_resolver(legs_spec)
```

Comment:
You can do `schema = DeclarativeSchema(STANDARD_RESOLVER + [legs_spec])` and skip the `add_resovler()`

Reply:
fixed!